src/assets/styles.js

```diff
@@ -130,6 +130,56 @@ export default StyleSheet.create({
   CCcontent: {
     height: 600,
   },
+  //Criminal Code PARTS/SECTION Screen Styling - Sprint 4
```

Comment:
I'm gonna feel bad for our designer that we will dump this on 😄 

There's probably so much overlapping styling that we've all put in that could be cleaned up 🤣

Reply:
Absolutely!  I am trying to remove those I have done previously but I think I missed a couple.  That's why I decided to delineate as other have done with comments.  And I scour the file for things I can reuse as much as I can.